analyzer/codechecker_analyzer/analyzers/cppcheck/analyzer.py

```diff
@@ -361,9 +361,10 @@ def version_compatible(cls, configured_binary, environ):
         # The analyzer version should be above 1.80 because '--plist-output'
         # argument was introduced in this release.
         if analyzer_version >= StrictVersion("1.80"):
-            return True
+            return None
 
-        return False
+        return "CppCheck binary found is too old at " \
```

Comment:
Does not print an error anymore. #4041 should solve not being able to give a recent enough gcc to CodeChecker. I wouldn't rush to print the binary location under the same breath -- not because thats not a good idea, I just wouldn't increase the size of this patch or the burden on us before release.